salt/_pillar/metalk8s.py

```diff
@@ -49,12 +51,42 @@ def _load_ca(config_data):
     }
 
 
+def _get_endpoints(config="/etc/kubernetes/admin.conf",
+                   context="kubernetes-admin@kubernetes",
+                   namespace="kube-system"):
```

Comment:
Making `namespace` an input argument, but `service` not, isn't very useful.

No need to make any of this configurable, IMHO. We know what this `ext_pillar` is supposed to provide. If more, then code changes are required.

Reply:
will fix